tests/test_never_type/test_runtime.py

```diff
@@ -0,0 +1,30 @@
+from pytest import mark, raises
+
+from basedtyping import Never, issubform
+
+
+def test_isinstance() -> None:
+    # https://github.com/python/mypy/issues/11903
+    assert not isinstance(  # type:ignore[misc]
```

Comment:
This needs to use pytest.raises right?

Reply:
i made it `xfail` because we want it to work once #22  is done